website_snippet_barcode/static/tests/js/website_snippet_barcode.js

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2017 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). */
+
+odoo.define_section('website_snippet_barcode', ['module_name.ExportedObject'], function(test) {
+    "use strict";
+
+    test('It should demonstrate a PhantomJS test for web (backend)',
```

Comment:
This is for backend test; you'll need a website tour instead

Reply:
Thanks. I'll add the tour next.